.gitignore

```diff
@@ -37,7 +37,3 @@
 # Ignore cache files
 **/.atmos/cache.yaml
 **/cache.*.txt
-
-# Ignore components vendored during tests
-tests/fixtures/scenarios/vendor/components/**
-examples/demo-vendoring/components/**
```

Comment:
Comment for others: 
- Files ignored with .gitignore do not appear when querying for untracked files in go-git. While they are not tracked by Git, being listed in .gitignore makes them behave as if they are, preventing their detection as untracked. We rely on untracked files to identify files for cleanup during tests, so this behavior interferes with that process.